napari/_qt/qt_viewer.py

```diff
@@ -6,20 +6,23 @@
 from qtpy.QtCore import QCoreApplication, Qt, QSize
 from qtpy.QtWidgets import QWidget, QVBoxLayout, QSplitter, QFileDialog
 from qtpy.QtGui import QCursor, QPixmap
+from vispy.scene import SceneCanvas, PanZoomCamera, TurntableCamera
 from qtpy import API_NAME
-from vispy.scene import SceneCanvas, PanZoomCamera
 from vispy.app import use_app
 
+from napari.components import Dims
```

Comment:
you shouldn't need to import `Dims` to get this working, I'll put more detail below